packages/transmute-cli/components/ansible/k8s-install-kong.yml

```diff
@@ -5,7 +5,7 @@
   vars:
     ipfs_started: no
     ganache_started: no
-    KONG_PROXY_PORT: 32443
+    kong_proxy_port: 32443
```

Comment:
Changing all instances of `KONG_PROXY_PORT` to `kong_proxy_port` to adhere to Ansible style best practices and to avoid confusion with shell environment variables.